src/background/actions/action-payloads.ts

```diff
@@ -4,7 +4,7 @@ import * as TelemetryEvents from '../../common/telemetry-events';
 import { BaseTelemetryData, TelemetryData, ToggleTelemetryData } from '../../common/telemetry-events';
 import { DetailsViewPivotType } from '../../common/types/details-view-pivot-type';
 import { ManualTestStatus } from '../../common/types/manual-test-status';
-import { BugServiceProperties } from '../../common/types/store-data/user-configuration-store';
+import { IssueServiceProperties } from '../../common/types/store-data/user-configuration-store';
```

Comment:
In my opinion, we should be renaming this property to `IssueFillingServiceProperties`.